cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,58 @@
 /// <reference types='cypress' />
+const { generateRegistrationInput } =
+require('../support/generateRegistrationInput');
 
 describe('Student Registration page', () => {
-  before(() => {
-
+  beforeEach(() => {
   });
 
-  it('', () => {
+  it('should reg new user', () => {
+    const {
+      firstName,
+      lastName,
+      email,
+      number,
+      address,
+      subject
+    } = generateRegistrationInput();
```

Comment:
The `subject` field is generated using `faker.lorem.sentences().slice(2, 11)`, which might not produce a valid subject input for the form. Consider using a more appropriate method to generate a single subject name or keyword.